packages/flutter_tools/templates/list_detail_app/test/implementation_test.dart.test.tmpl

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
```

Comment:
looks like your .golden.tmpl files are empty, btw.

(in any case, please don't commit binary files to this repo.)

Reply:
Thanks -- that was intentional for the current setup, they were meant to be placeholder files that are replaced by binaries from the template_image_assets repo. Sounds like this will change now that I'm aware of Flutter Gold :)